backend/infrahub/core/diff/model/path.py

```diff
@@ -447,7 +447,7 @@ def __hash__(self) -> int:
         return hash(self.uuid)
 
     @property
-    def time_range(self) -> Interval:
+    def time_range(self) -> TimeDelta:
```

Comment:
As per the updates in the SDK the time_range here and on line 420 will raise a deprecation warning due to us keeping the code in the function intact.

```python
return self.to_time.obj - self.from_time.obj
```
We don't want to use `._obj` instead but perhaps we need some other method. Or some dundermethods on our own class so that it's possible to instead run:

```python
return self.to_time - self.from_time
```

Reply:
Good catch, I fixed it